vllm/core/block_manager_v1.py

```diff
@@ -659,7 +661,7 @@ def access_all_blocks_in_seq(
             # in this step.
             block_table = self.block_tables[seq.seq_id]
             for block in block_table:
-                block.last_accessed = access_time
+                block.last_accessed = int(access_time)
```

Comment:
I think `last_accessed` should be a `float` instead of a `int`, rather than the other way round.